CHANGES.rst

```diff
@@ -11,6 +11,65 @@ Changes
 -------
 
 - Drop support for Python 3 versions older than Python 3.6. (#341)
+- Re-export CorePlugin in envisage.api (#332)
+- reate and fill plugin subpackage api modules (#323)
+- Add relevant classes to envisage.ui.tasks.api (#322)
+- Make TasksApplication.gui expect an IGUI interface, not a GUI instance (#301)
+- Deprecate safeweakref and replace its uses (#275)
+
+Fixes
+-----
+
+- Fix unregister_service ValueError bug (#345)
+- Fix the MOTD example (#319)
+- Fix the Hello_World example (#318)
+- Fix the attractors tasks application example (#317)
+
+Features
+--------
+
+- Add a new shell click command (#305)
+
+Documentation
+-------------
+
+- Setup intersphinx in docs (#343)
+- Add documentation for envisage APIs (#340)
+- Use jinja templates for API documentation (#339)
+- Improve API docs : document traits (#334)
+- Rebuild documentation, mostly to fix search functionality (#290)
+
+Removals
+--------
+
+- Remove single_project (#331)
+- Remove class_load_hooks and ClassLoadHook (#321)
+
+Tests
+-----
+
+- Add tests for ExtensionRegistry getters (#349)
+- Add tests to demonstrate behaviour when mutating extension point directly
+  (#346)
+- Use mixin instead of having ProviderExtensionRegistryTestCase inherit from
+  ExtensionRegistryTestCase (#335)
+- TST: Switch on default warning flag for CI test command (#326)
+- Add test eggs for Python 3.9; remove eggs for Python 2.7 (#289)
+
+Build
+-----
+
+- Add apptools to source_dependencies in etstool.py (#348)
+- Update setup.py to allow prerelease version (#344)
+- Add wx as being supported in etstool, add it back to CI, and test against
+  wxPython v4.x (#336)
+- Use edm 3.0.1 instead of 2.0.0 (#297)
+- Don't report code coverage (#288)
+- Fix CI failures on Linux, Windows (#287)
```

Comment:
Reading this out-of-context I thought there were library related bugs being fixed, but this was related to CI setup.
```suggestion
- Fix CI setup on Linux, Windows (#287)
```